config/application.rb

```diff
@@ -22,6 +22,9 @@ class Application < Rails::Application
     config.menu_entries = [
       { caption: 'Welcome', icon: 'home', target: '/'}
     ]
+    config.lasso_run = "run"
+    config.lasso_commands = "nil"
+    config.lasso_commands_file = '/tmp/delete_resources_steps'
```

Comment:
Probably want to store this somewhere that's app specific. Otherwise there's no control if `/tmp` dir gets wiped out.

Reply:
OK, will update the path